pkg/providers/aws/provider_postgres.go

```diff
@@ -567,6 +581,20 @@ func (p *PostgresProvider) getRDSConfig(ctx context.Context, r *v1alpha1.Postgre
 	return rdsCreateConfig, rdsDeleteConfig, stratCfg, nil
 }
 
+func (p *PostgresProvider) isLastResource(ctx context.Context) (bool, error) {
```

Comment:
While this may never happen, I am just thinking of the case something outside of CRO's control gets provisioned in the CRO VPC, how would we handle this situation. We would error out and the un-install would never complete. Is this a behaviour we want?
Like above might be worth a discussion :thinking:

Reply:
Let's namespace the list to the watch namespace. I think this situation is unlikely to happen.

Reply:
agree, unlikely to happen. 

Good catch on reducing the list scope to the watch namespace

Reply:
@laurafitzgerald Unless there are any objections, we can go with that?

Reply:
@aidenkeating @ciaranRoche the check now accepts the watch namespace parameter. Updated the verification to include that as a check. 👍